forgettable-frontend/src/components/EncounterCardSummary/EncounterSummary.module.css

```diff
@@ -0,0 +1,62 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
```

Comment:
Don't think the import is necessary. Font has already been set to prioritise Poppins by default globally, thanks @rainasong <3